src/scss/includes/popup.scss

```diff
@@ -1,4 +1,8 @@
 .poi_popup__container {
+
+  position: absolute;
+  z-index: 6;
```

Comment:
I think we have a dedicated file for `z-index`es, you could eventually put it there to be in adequation with the purpose of this file 👍